src/components/MainVisualization.js

```diff
@@ -0,0 +1,15 @@
+import OverallStatsViz from './OverallStatsViz';
+import PatientStatsViz from './PatientStatsViz';
+import OutcomeStatsViz from './OutcomeStatsViz';
+
+function MainVisualization({ coverageData }) {
+  return (
+    <>
+      <OverallStatsViz coverageData={coverageData} />
+      <PatientStatsViz coverageData={coverageData} />
```

Comment:
![Screen Shot 2022-04-12 at 2 49 31 PM](https://user-images.githubusercontent.com/53443041/163032693-142ad860-4daa-4882-b9d0-877a64f29587.png)
I think we should probably only display each component here if a section with its corresponding `sectionId` is present in the `coverageData` object, or default the section to 0%. If you remove the patient resource from the testBundle for instance, a `NaN` percentage is displayed. 

I'm not sure if we'd want to not list empty sections (if there's no resources in a section then the implementers probably aren't concerned with that segment of the spec) or output a 0% stat (to be completely accurate) but the current display does seem a bit jarring.

Reply:
oh we definitely want to make sure that the computation can handle scenarios where the incoming bundle doesn't have any corresponding resources. I'll take a look at this one.

Reply:
This should be resolved now.

Reply:
Lovely!